Renderer.js

```diff
@@ -0,0 +1,36 @@
+class Renderer {
+    renderUserData(userInfo){
+        const source = $("#mainUser").html()
+        const template = Handlebars.compile(source)
+        const userDataHTML = template(userInfo)
+        $(".user-container").empty().append(userDataHTML)
+
+    }
+    
+    renderKanye(kq){
+        const source2 = $("#quote").html()
```

Comment:
No need to give different names to the variables since they are in different scopes (no need for `source2` can just be `source` and so on...)